composer.json

```diff
@@ -14,9 +14,9 @@
         "phayes/geophp": "^1.2"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.15",
         "phing/phing": "^2.16",
         "phpunit/phpunit": "^5.7",
-        "squizlabs/php_codesniffer": "^3.4"
+        "squizlabs/php_codesniffer": "2.*",
```

Comment:
Why older version?

Reply:
Actually, my composer seems to be messing up totally, gonna check what is happening